Azure-Samples/openai
# AOAI Search Demo release 1.1.0

## Purpose

This PR makes a new release for the AOAI Search end-to-end demo. The updates for this release include:
* New vector search option for unstructured approach queries.
* Updates to both the deployment process and deployment README steps to accommodate vector search and to make the deployment process simpler.
* Updates to classifier prompt to include content safety guardrails.

## Does this introduce a breaking change?

```
[ x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ x ] Feature
[ x ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

1. Clone repo
2. Follow local and Azure deployment setup instructions on README.md, verify necessary resources are provisioned successfully and the app runs as expected with no issues, both locally and on Azure.
